terraform/multipass/nginx.tftpl

```diff
@@ -0,0 +1,19 @@
+worker_processes 1;# Core count
+events {
+    worker_connections  1024;# Connections per sec per core
+}
+
+http {
+    upstream avax {
+        %{ for ip in validators[*].ipv4 ~}
+        server ${ip}:9650;
+        %{ endfor ~}
+    }
+    server {
+        listen 80;
```

Comment:
IMO it would make more sense to use the `9650` port for redirection instead of `80`.

Reply:
@leopaul36 apparently you already discussed this. I find it weird to try to bind on port `80` of the user given that it might be used by a lot of other services. Using `9650` seems more logical to access Avalanche nodes.

Reply:
![image](https://github.com/AshAvalanche/ansible-avalanche-getting-started/assets/11584800/7df043ba-6f18-492f-9d02-7b7ff64e51c3)

Reply:
From wallet.avax.network

Reply:
Let's merge like this for now, we can always change later if we have issues with port 80